server.js

```diff
@@ -362,6 +363,7 @@ Server.get_smtp_server = function (host, port, inactivity_timeout, done) {
     }
     else {
         server = tls_socket.createServer(onConnect);
+        server.has_tls = false;
```

Comment:
Is it still not a tls server at this point?

Reply:
tls_socket is a socket that can potentially upgrade to TLS (if and when starttls gets issued). doesn't mean it's already using TLS. in the above case, it's not upgrade to tls (yet).